lib/xclarity_client/unmanage_request_management.rb

```diff
@@ -0,0 +1,41 @@
+require 'json'
+
+module XClarityClient
+  class UnmanageRequestManagement < XClarityBase
+
+    include XClarityClient::ManagementMixin
+
+    def initialize(conf)
+      @conf = conf
+      super(conf, UnmanageRequest::BASE_URI)
+    end
+
+    def unmanage_discovered_devices(endpoints, force)
+      deploy_hash = {}
+      deploy_hash.merge!({:endpoints => endpoints})  
+      if force.eql? "true" or force.eql? "True"
```

Comment:
Why test upper and lower case here? You can just do force.downcase.eql? "true" and remove the second statement.